app/AndroidManifest.xml

```diff
@@ -64,6 +64,7 @@
         android:label="@string/application_name"
         android:hardwareAccelerated="true"
         android:usesCleartextTraffic="true"
+        android:requestLegacyExternalStorage="true"
```

Comment:
Read here for more information :: https://developer.android.com/training/data-storage#scoped-storage
If not enabled, this method throws error https://github.com/dimagi/commcare-android/blob/master/app/src/org/commcare/models/ODKStorage.java#L22

Reply:
For Android 10 adding this sounds good. Though we should add Soft asserts log  statements to the code paths that we identified that were still accesing code paths. So that we know in future if they are indeed getting hit or not.